examples/random/src/main.rs

```diff
@@ -2,14 +2,15 @@
 #![no_std]
 #![feature(used_with_arg)]
 
-use ariel_os::debug::log::*;
+use ariel_os::{asynch::Spawner, debug::log::*};
+use rand::Rng as _;
 
-#[ariel_os::thread(autostart)]
-fn main() {
-    use rand::Rng as _;
+#[ariel_os::spawner(autostart)]
```

Comment:
I think we should use a regular task here.

Reply:
I'm thinking educational, examples should be as uniform and consistent as possible. If the random example differs from hello-world only in the random use, perfect. if it suddenly uses the spawner mechanism, it is "too different".
I'm OK with the extra feature, as that is boilerplate we'll explain away once, centrally. :)

Reply:
Looks to me like that's done.